pkg/morph/client/constructor.go

```diff
@@ -106,101 +106,90 @@ func New(key *keys.PrivateKey, opts ...Option) (*Client, error) {
 		opt(cfg)
 	}
 
-	cli := &Client{
-		cache:      newClientCache(),
-		logger:     cfg.logger,
-		acc:        acc,
-		accAddr:    accAddr,
-		cfg:        *cfg,
-		switchLock: &sync.RWMutex{},
-		closeChan:  make(chan struct{}),
-		notifyChan: make(chan *state.ContainedNotificationEvent),
-		blockChan:  make(chan *block.Block),
-		notaryChan: make(chan *result.NotaryRequestEvent),
-		subs: subscriptions{
-			subscribedEvents:       make(map[util.Uint160]struct{}),
-			subscribedNotaryEvents: make(map[util.Uint160]struct{}),
-			subscribedToNewBlocks:  false,
-
-			curNotifyChan: make(chan *state.ContainedNotificationEvent),
-			curBlockChan:  make(chan *block.Block),
-			curNotaryChan: make(chan *result.NotaryRequestEvent),
-		},
-	}
-
-	var err error
-	var act *actor.Actor
+	var (
```

Comment:
even here? more parentheses, more nesting, more lines

Reply:
I can make add even more of them into this `var ()` block for you. But hey, we're defining some variables, `var ()` is exactly for that and it's preferred over a series of `var ... var ... var ...`.

Reply:
as always, i prefer `:=`